rocketpy/Environment.py

```diff
@@ -3221,6 +3221,117 @@ def allInfo(self):
 
         return None
 
+    def allPlotInfoReturned(self):
+        """Returns a dictionary with all plot information available about the Environment.
+
+        Parameters
+        ----------
+        None
+
+        Returns
+        ------
+        plotInfo : Dict
+            Dict of data relevant to plot externally
+        """
+        grid = np.linspace(self.elevation, self.maxExpectedHeight)
```

Comment:
`np.linspace` returns 50 evenly spaced samples by default. Is this good enough for all cases? 

These Functions, such as `windSpeed` and `density` can sometimes be defined as arrays, meaning they are already sampled data. In such cases, could the `grid` be just `windSpeed.source[:, 0]`?